report/clients/standalone-flow.tsx

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
probably makes sense in `report/clients/flow/*`, especially if it's going to have its own tsconfig, eslint rules, etc

Reply:
maybe we should do a new folder `/flow`? toplevel.

Reply:
> maybe we should do a new folder `/flow`? toplevel

yeah, it's different enough that makes sense too

Reply:
I like toplevel

Reply:
in that case I'd prefer `flow-report/` fyi